### PR TITLE
feat(InstantSearch.js) Add geo-search-tourism sample

### PR DESCRIPTION
**Pull request by clemfromspace**
Moving this sample from https://github.com/algolia/instantsearch.js/tree/master/examples/tourism to this repository, and adding a small README to it.

### CODE REVIEW

File: InstantSearch.js/geo-search-tourism/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+Alternatively, you may use Yarn:
+
+```
+yarn
+yarn start
+```
+
+Open http://localhost:3000 to see your app.
```
Comment (Haroenv) [inline InstantSearch.js/geo-search-tourism/README.md:46]:
```suggestion
Open <http://localhost:3000> to see your app.
```

File: InstantSearch.js/geo-search-tourism/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Instantsearch tourism sample
+
+This sample shows how to leverage [Algolia Geo Location](https://www.algolia.com/doc/guides/managing-results/refine-results/geolocation/) with [InstantSearch's geoSearch widget](https://www.algolia.com/doc/api-reference/widgets/geo-search/js/#widget-param-googlereference) with [Google Maps](https://developers.google.com/maps).
+
+<img src="InstantSearch.js/geo-search-tourism/capture.png?raw=true" alt="A capture of the Algolia InstantSearch tourism demo" align="center">
```
Comment (Haroenv) [inline InstantSearch.js/geo-search-tourism/README.md:5]: why the raw=true?

also align needs to be on a wrapping `p` I think?

Comment (Haroenv) [reply InstantSearch.js/geo-search-tourism/README.md:5]:
```suggestion
<img src="capture.png?raw=true" alt="A capture of the Algolia InstantSearch tourism demo" align="center">
```

relative link :)

File: InstantSearch.js/geo-search-tourism/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+## Features
+
+This sample showcase the following features:
+
+- 🗺️ InstantSearch [geoSearch widget](https://www.algolia.com/doc/api-reference/widgets/geo-search/js/#widget-param-googlereference).
+- 📍 Algolia [aroundLatLngViaIP](https://www.algolia.com/doc/api-reference/api-parameters/aroundLatLngViaIP/) API parameter.
+
+## Demo
+
+[Access the demo](https://react-instantsearch.netlify.app/examples/tourism/)
```
Comment (Haroenv) [inline InstantSearch.js/geo-search-tourism/README.md:16]:
```suggestion
[Access the demo](https://codesandbox.io/s/github/algolia/doc-code-samples/tree/master/InstantSearch.js/geo-search-tourism)
```

**Review by Haroenv — COMMENTED** [review]
👌

File: InstantSearch.js/geo-search-tourism/README.md (hunk 2 above)
Comment (clemfromspace) [reply InstantSearch.js/geo-search-tourism/README.md:5]: 👌

**Review by Haroenv — APPROVED** [review]

**Pull request merged**